packages/redot-parse/test.js

```diff
@@ -0,0 +1,26 @@
+import {test} from 'node:test'
+import * as assert from 'node:assert/strict'
```

Comment:
```suggestion
import assert from 'node:assert/strict'
```

Is a bit clearer to me. I think that’s also what all the Node docs use?

Reply:
good idea, updated